notebooks/src/el-paso.ipynb

```diff
@@ -538,18 +558,44 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "@ergo.mem\n",
+    "def peak_hospitalized():\n",
```

Comment:
I think it would be slightly cleaner if this function were just `peak_hospitalization_date` and in `max_30d_hospital_confirmed_for_peak` we look up the hospitalization count using `hospital_confirmed_for_date(peak_hospitalization_date())`

Reply:
yeah that does seem better now that I think about it